apps/intellij/src/main/kotlin/dev/nx/console/generate/NxGeneratorSearchEvrywhereContributor.kt

```diff
@@ -0,0 +1,121 @@
+package dev.nx.console.generate
+
+import com.intellij.icons.AllIcons
+import com.intellij.ide.actions.searcheverywhere.SearchEverywhereContributor
+import com.intellij.ide.actions.searcheverywhere.SearchEverywhereContributorFactory
+import com.intellij.openapi.actionSystem.ActionPlaces
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.components.service
+import com.intellij.openapi.fileEditor.FileEditorManager
+import com.intellij.openapi.progress.ProgressIndicator
+import com.intellij.openapi.progress.ProgressManager
+import com.intellij.openapi.util.text.StringUtil
+import com.intellij.psi.codeStyle.NameUtil
+import com.intellij.util.Processor
+import dev.nx.console.generate.ui.DefaultNxGenerateUiFile
+import dev.nx.console.generate.ui.NxGeneratorListCellRenderer
+import dev.nx.console.models.NxGenerator
+import dev.nx.console.models.NxGeneratorOption
+import dev.nx.console.nxls.server.requests.NxGeneratorOptionsRequestOptions
+import dev.nx.console.services.NxlsService
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentMap
+import javax.swing.ListCellRenderer
+import kotlinx.coroutines.runBlocking
+
+class NxGeneratorSearchEvrywhereContributor : SearchEverywhereContributorFactory<NxGenerator> {
```

Comment:
Shouldn't this be `NxGeneratorSearchEverywhereContributorFactory`? Also there is an e missing in everywhere